src/webgpu/api/validation/state/device_lost/destroy.spec.ts

```diff
@@ -0,0 +1,543 @@
+  `
+  )
+  .unimplemented();
+
+g.test('command,copyTextureToBuffer,uncompressed_format')
+  .desc(
+    `
+Tests copyTextureToBuffer command with various uncompressed formats on destroyed device fails.
+  - Tests that finishing encoding fails on destroyed device
+  - Tests that submitting command fails on destroyed device
+  `
+  )
+  .unimplemented();
+
+g.test('command,finish,copyTextureToBuffer,compressed_format')
+  .desc(
+    `
+Tests copyTextureToBuffer command with various compressed formats on destroyed device fails.
+  - Tests that finishing encoding fails on destroyed device
+  - Tests that submitting command fails on destroyed device
+  `
+  )
+  .unimplemented();
+
+g.test('command,copyTextureToTexture,uncompressed_format')
+  .desc(
+    `
+Tests copyTextureToTexture command with various uncompressed formats on destroyed device fails.
+  - Tests that finishing encoding fails on destroyed device
+  - Tests that submitting command fails on destroyed device
+  `
+  )
+  .unimplemented();
+
+g.test('command,copyTextureToTexture,compressed_format')
+  .desc(
+    `
+Tests copyTextureToTexture command with various compressed formats on destroyed device fails
+  - Tests that finishing encoding fails on destroyed device
+  - Tests that submitting command fails on destroyed device
+  `
+  )
+  .unimplemented();
+
+g.test('command,clearBuffer')
+  .desc(
+    `
+Tests encoding and finishing a clearBuffer command on destroyed device fails.
+  - Tests that finishing encoding fails on destroyed device
+  - Tests that submitting command fails on destroyed device
+  `
+  )
+  .unimplemented();
+
+g.test('command,writeTimestamp')
+  .desc(
+    `
+Tests encoding and finishing a writeTimestamp command on destroyed device fails.
+  - Tests that finishing encoding fails on destroyed device
+  - Tests that submitting command fails on destroyed device
+  `
+  )
+  .unimplemented();
+
+g.test('command,resolveQuerySet')
+  .desc(
+    `
+Tests encoding and finishing a resolveQuerySet command on destroyed device fails.
+  - Tests that finishing encoding fails on destroyed device
+  - Tests that submitting command fails on destroyed device
+  `
+  )
+  .unimplemented();
+
+g.test('command,computePass,dispatch')
+  .desc(
+    `
+Tests encoding and dispatching a simple valid compute pass on destroyed device fails.
+  - Binds valid pipeline and bindgroups, then dispatches
+  - Tests that finishing encoding fails on destroyed device
+  - Tests that submitting command fails on destroyed device
+  `
+  )
+  .unimplemented();
+
+g.test('command,renderPass,draw')
+  .desc(
+    `
+Tests encoding and finishing a simple valid render pass on destroyed device fails.
+  - Binds valid pipeline and bindgroups, then draws
+  - Tests that finishing encoding fails on destroyed device
+  - Tests that submitting command fails on destroyed device
+  `
+  )
+  .unimplemented();
+
+g.test('command,renderPass,renderBundle')
+  .desc(
+    `
```

Comment:
maybe we need queue.writeBuffer, queue.writeTextue, and queue.copyExternalImageToTexture as well

Reply:
Oops, I actually had those but I accidentally deleted them in my splitting. Good catch!